galleon-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/narayana/rts/lra-coordinator/main/module.xml

```diff
@@ -44,6 +44,7 @@
     <module name="jakarta.servlet.api"/>
     <module name="org.eclipse.microprofile.openapi.api"/>
     <module name="io.smallrye.fault-tolerance"/>
+    <module name="io.smallrye.config" optional="true"/>
```

Comment:
This being optional implies that you were thinking in terms of the layer dependency being optional as well.

This being added at all implies that something was missing before, i.e. we have a bug. In that case adding some kind of regression test would be good.